src/dotnet/commands/dotnet-help/xlf/LocalizableStrings.cs.xlf

```diff
@@ -158,7 +158,7 @@
         <note />
       </trans-unit>
       <trans-unit id="CommandDoesNotExist">
-        <source>Specified command '{0}' is not a valid SDK command. Please specify a valid SDK command. For more information, run dotnet help.</source>
+        <source>Specified command '{0}' is not a valid SDK command. Specify a valid SDK command. For more information, run dotnet help.</source>
```

Comment:
The xlf files should never bemanually changed. Only the source ../LocalizableStrings.resx. Then a local build should update them.

This has the target in the wrong state. It should have switched to "needs-review-translation" by the update.

Revert all .xlf files and run build.cmd

Reply:
So the instructions I had added for the xlf files on the developer guide, should be added back @nguerrera?

Reply:
We can add them back, but you just need to run the build script after changing resx files.

Reply:
So how did @nguerrera get the needs-review-translation in the localized xlf files? If I just make changes to the english xlf and create a PR, nothing happens. The instruction should be there (plain build or build with the flag).

Reply:
@mairaw The rule is: don't ever manually edit an xlf. Edit the corresponding `.resx` and let the build propagate the changes to the xlf (with metadata like `needs-review-translation` here, so that the loc team gets the right updates and can give back the final translations).

Reply:
Ah ok,  now I remembered how this worked. Thanks guys!
But you still need to run the build with the flags right?
I remember I added that info after struggling with this in a PR.

Reply:
No flags anymore. Edit resx and build as usual when editing cs. We can still call this out in docs.

Reply:
Ok, I'll create a PR for that. Since I don't do that on a regular basis, I tend to forget 😄